src/test/java/uk/gov/hmcts/reform/professionalapi/entities/ProfessionalUserTest.java

```diff
@@ -33,7 +34,8 @@ public void creates_professional_user_correctly() {
         assertThat(professionalUser.getOrganisation()).isEqualTo(organisation);
         assertThat(professionalUser.getLastUpdated()).isNull();
         assertThat(professionalUser.getCreated()).isNull();
-        assertThat(professionalUser.getUserIdentifier()).isNull();
+        assertThat(professionalUser.getUserIdentifier()).isNotNull();
+        assertThat(professionalUser.getUserIdentifier().toString().length()).isEqualTo(36);
```

Comment:
Can we name the magic numbers to be meaningful e.g.  private static final int LENGTH_OF_UUID;

Reply:
Done